api/server/server.go

```diff
@@ -321,8 +321,11 @@ func getImagesViz(eng *engine.Engine, version version.Version, w http.ResponseWr
 
 func getInfo(eng *engine.Engine, version version.Version, w http.ResponseWriter, r *http.Request, vars map[string]string) error {
 	w.Header().Set("Content-Type", "application/json")
-	eng.ServeHTTP(w, r)
-	return nil
+	if env, err := getDaemon(eng).Info(); err != nil {
```

Comment:
It is better to use simple form without else. Also it is recommended by google:
https://code.google.com/p/go-wiki/wiki/CodeReviewComments#Indent_Error_Flow